### ISSUE TITLE
I already downloaded the openwebtext  , but how can i load the openwebtext.tar.xz from my disk?

### ISSUE THREAD

**Issue by cabinhou** [karpathy/nanoGPT#139]
where is the openwebtext file  cache directory？

`# takes 54GB in huggingface .cache dir, about 8M documents (8,013,769)
dataset = load_dataset("openwebtext")

# owt by default only contains the 'train' split, so create a test split
split_dataset = dataset["train"].train_test_split(test_size=0.0005, seed=2357, shuffle=True)
split_dataset['val'] = split_dataset.pop('test') `

**Comment by HadesGray** [comment]
/Users/hevil/.cache/huggingface/modules/datasets_modules/datasets/openwebtext/85b3ae7051d2d72e7c5fdf6dfb462603aaa26e9ed506202bf3a24d261c6c40a1/openwebtext.py

the line:
dl_dir = dl_manager.download_and_extract(_URL)

change "_URL" to your local path

**Comment by cabinhou** [comment]
Thank you very much！I  can finally finish this step，use this sh command ：sudo find / | grep openwebtext.py ，then edit the openwebtext.py file。

**Comment by clclclaiggg** [comment]
hello，Where do I get openwebtext.py？

**Comment by madocoin** [comment]
متى ارسلت هذا البريد

**Comment by madocoin** [comment]
I ANDERSTAND but.................. 2021
